pyomo/contrib/interior_point/linalg/scipy_interface.py

```diff
@@ -0,0 +1,67 @@
+from .base_linear_solver_interface import LinearSolverInterface
+from .results import LinearSolverStatus, LinearSolverResults
+from scipy.sparse.linalg import splu
+from scipy.linalg import eigvals
+from scipy.sparse import isspmatrix_csc
+from pyomo.contrib.pynumero.sparse.block_vector import BlockVector
+import logging
+import numpy as np
+
+
+class ScipyInterface(LinearSolverInterface):
+    def __init__(self, compute_inertia=False):
+        self._lu = None
```

Comment:
I think we should add a comment here to indicate that compute_inertia could make things pretty slow for large problems (computing the eigenvalues can be very expensive)

Reply:
Good point.